Services/AssessmentQuestion/interfaces/interface.ilAsqQuestionAuthoring.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/* Copyright (c) 1998-2013 ILIAS open source, Extended GPL, see docs/LICENSE */
+
+/**
+ * Interface ilAsqQuestionAuthoring
+ *
+ * @author    Björn Heyser <info@bjoernheyser.de>
+ * @version    $Id$
+ *
+ * @package    Services/AssessmentQuestion
+ */
+interface ilAsqQuestionAuthoring
+{
+	/**
+	 * Execute Command
+	 */
+	public function executeCommand();
+	
+	/**
+	 * @param ilAsqQuestion $question
+	 * @return void
+	 */
+	public function setQuestion(ilAsqQuestion $question);
+
+	/**
+	 * @param ilQuestionChangeListener $listener
+	 */
+	public function addQuestionChangeListener(ilQuestionChangeListener $listener);
+	
+	/**
+	 * @param object $a_object
+	 * @param string $a_method
+	 * @param mixed $a_parameters
+	 */
+	public function addNewIdListener($a_object, $a_method, $a_parameters = "");
```

Comment:
What's this?

Reply:
a legacy depency that i cannot get rid of without reworking the presentation concepts with all the technical issues behind to come to a single question presentation implementation.